src/Interpreters/FillingRow.h

```diff
@@ -15,16 +15,20 @@ bool equals(const Field & lhs, const Field & rhs);
  */
 class FillingRow
 {
+    std::optional<Field> doJump(const FillColumnDescription & descr, size_t column_ind);
+    std::optional<Field> doLongJump(const FillColumnDescription & descr, size_t column_ind, const Field & to);
```

Comment:
Comment here could help explain the purpose of methods and clarify the distinction between a regular and long jump

Reply:
Did that, thanks!